src/main/java/net/mcreator/minecraft/MinecraftImageGenerator.java

```diff
@@ -1435,6 +1435,9 @@ public static BufferedImage generateTagPreviewPicture(String type) {
 				case "Entities" -> ImageUtils.toBufferedImage(
 						ImageUtils.colorize(UIRES.get("mod_preview_bases.tag"), Dependency.getColor("entity"), false)
 								.getImage());
+				case "Fluids" -> ImageUtils.toBufferedImage(
+						ImageUtils.colorize(UIRES.get("mod_preview_bases.tag"), Dependency.getColor("fluid"), false)
```

Comment:
`fluid` is nor a variable type nor a dependency inside the switch statement, so the returned color is white. Is it what you want? I'm not sure it's the best option to put it white. I think another color should be used.

Reply:
What then? Is there a way to get a mix of the main color of the fluids used in the tag?